src/core/main.cpp

```diff
@@ -137,6 +142,14 @@ int main( int argc, char * * argv )
 		}
 	}
 
+#ifndef LMMS_WIN32
```

Comment:
Its not a new thing, ever since lmms had travis build osx, my local repo fails every time, hence considering paying for osx on my reapo if it's cheep enough, according to my inbox, travis failed is my new best friend :) lol